### PR TITLE
New package: Smash++

### PR DESCRIPTION
**Pull request by smortezah**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

 * `@BiocondaBot please update` will cause the BiocondaBot to merge the master branch into a PR
 * `@BiocondaBot please add label` will add the `please review & merge` label.
 * `@BiocondaBot please fetch artifacts` will post links to packages and docker containers built by the CI system. You can use this to test packages locally before merging.

For members of the Bioconda project, the following command is also available:

 * `@BiocondaBot please merge` will cause packages/containers to be uploaded and a PR merged. Someone must approve a PR first! This has the benefit of not wasting CI build time required by manually merging PRs.

If you have questions, please post them in gitter or ping `@bioconda/core` in a comment (if you are not able to directly ping `@bioconda/core` then the bot will repost your comment and enable pinging).

### CODE REVIEW

File: meta.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{% set version = "19.12" %}
```
Comment (bgruening) [inline meta.yaml]: can you please use 4 space intend ...

File: meta.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+package:
+        name: smashpp
+        version: {{ version }}
+
+build:
+        number: 0
+
+source:
+        url: https://github.com/smortezah/smashpp/archive/v{{ version }}.tar.gz
```
Comment (bgruening) [inline meta.yaml]: please include a sha256 checksum

File: meta.yaml (hunk 1 above)
Comment (smortezah) [reply meta.yaml]: done

File: build.sh [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+export C_INCLUDE_PATH=$C_INCLUDE_PATH:${PREFIX}/include
+#export CPLUS_INCLUDE_PATH=$CPLUS_INCLUDE_PATH:${PREFIX}/include
+export LIBRARY_PATH=$LIBRARY_PATH:${PREFIX}/lib
+
+rm -fr build
+mkdir -p $PREFIX/build
+cd $PREFIX/build
+cmake ../src
+make -j8
+mv smashpp ..
```
Comment (bgruening) [inline build.sh]: Should $PREFIX/build be removed?
Normally we don't create the build dir under PREFIX but in the CWD, maybe that is also an alternative.

**Comment by bgruening** [comment]
@BiocondaBot please merge

**Comment by bgruening** [comment]
Thanks @smortezah !

**Comment by smortezah** [comment]
> I received an error uploading the build artifacts or merging the PR!

What should be done, now?

**Pull request merged**